raiden/raiden_service.py

```diff
@@ -539,6 +529,33 @@ def _start_transport(self, chain_state: ChainState) -> None:
             if neighbour != ConnectionManager.BOOTSTRAP_ADDR:
                 self.start_health_check_for(neighbour)
 
+    def _prepare_and_execute_alarm_first_run(self, from_block) -> None:
+        """Prepares the alarm task callback and executes its first run
+
+        Complete the first_run of the alarm task and synchronize with the
+        blockchain since the last run.
+
+         Notes about setup order:
+         - The filters must be polled after the node state has been primed,
+           otherwise the state changes won't have effect.
+         - The alarm must complete its first run before the transport is started,
```

Comment:
could you please add asserts for these two comments? Like so:

https://github.com/raiden-network/raiden/blob/c00eda0a89d2ff76b36d48b1f6da72f90fb9c422/raiden/raiden_service.py#L860-L861